lib/miq_automation_engine/service_models/miq_ae_service_metric_zones.rb

```diff
@@ -0,0 +1,4 @@
+module MiqAeMethodService
+  class MiqAeServiceMetricZones < MiqAeServiceModelBase
```

Comment:
I have no idea what that was therefore I will go out on a limb and suggest it isn't necessary in the service model.  I'm removing it.